src/assets/i18n/en.json

```diff
@@ -767,6 +768,14 @@
       "settings": {
         "fixedURLforOCPP": "From OCPP 1.6 on, the URL is fixed by the server",
         "callbackURLforOCPP": "Callback URL to send HTTP request to the charging station"
+      },
+      "enableManualConfiguration":{
+        "title": "Enable manual configuration",
+        "confirm": "If you enable the manual configuration, all charge points will be deleted and the technical configuration will not be taken from the template anymore. After saving the charging station will perform a reboot. Do you want to continue?"
```

Comment:
I don't remember why we needed a reboot here, maybe to go through the code that will init the params?
We can just remove the charge points and set the connectors info and let the user perform changes and save without rebooting I think.
On the contrary, if you put back the flag to false, a reboot will reajust everything.
Maybe we need a meeting.

So the message would be that:
"If you enable the manual configuration, the charging station will not be adjusted with the automatic configuration template anymore. Do you want to continue?"

Next how this charger will behave with the smart charging?